commands/cock.js

```diff
@@ -11,8 +12,21 @@ module.exports = {
         .setRequired(true)
     ),
   async execute(interaction) {
-    await interaction.reply(
-      `${interaction.options.getUser("user")} GOT A COCK ON EM!!!`
-    );
+    inch = getRandomInt(1,13)
+    const user = interaction.options.getUser("user")
+    if (user.id == 687842330006519880){
+      await interaction.reply(
+        `${interaction.options.getUser("user")} does not in fact have a cock on them, sadge :( `
+      );
+    } else if (user.id == 553792724373864448){
+      await interaction.reply(
+      `${interaction.options.getUser("user")} might have a cock on him if he ever coded, sadge :( `
+      )
+    } else {
+      await interaction.reply(
+        `${interaction.options.getUser("user")} GOT A ${inch} INCH COCK ON EM!!!`
```

Comment:
Nit picky, but could sanitize for `a` vs `an`: Got an 8 inch -> or Got a 7 inch.